tf2/src/static_cache.cpp

```diff
@@ -57,6 +58,8 @@ unsigned tf2::StaticCache::getListLength() {return 1;}
 
 tf2::CompactFrameID tf2::StaticCache::getParent(tf2::TimePoint time, std::string * error_str)
 {
+  (void)error_str;
+  (void)time;
```

Comment:
Nit:

```suggestion
  (void)time;
  (void)error_str;
```

(the order of the parameters)

Reply:
Fixed https://github.com/ros2/geometry2/pull/322/commits/606b511024633a6ceca341fbe99598fdc93cf5e5.